Apps/Common/test/unit/Services/SupportServiceTests.cs

```diff
@@ -187,6 +196,58 @@ public void ShouldGetVerifications()
             Assert.Equal(2, actualResult.ResourcePayload.Count());
         }
 
+        /// <summary>
+        /// Tests SupportService can get a list of delegate owner by the delegate's PHN.
```

Comment:
"Tests SupportService can get a list of delegate owner by the delegate's PHN."
=>
"Tests SupportService can get a list of delegates by the dependent's PHN."